### PR TITLE
Cm readme inference update

### PR DESCRIPTION
**Pull request by anandhu-eng**

### CODE REVIEW

File: docs/submission/index.md [hunk 1]
```diff
@@ -3,70 +3,107 @@ hide:
   - toc
 ---
 
-If you follow the `cm run` commands under the individual model pages in the [benchmarks](../benchmarks/index.md) directory, all the valid results will get aggregated to the `cm cache` folder. Once all the results across all the modelsare ready you can use the following command to generate a valid submission tree compliant with the [MLPerf requirements](https://github.com/mlcommons/policies/blob/master/submission_rules.adoc#inference-1).
+=== "CM based benchmark"
+    If you have followed the `cm run` commands under the individual model pages in the [benchmarks](../index.md) directory, all the valid results will get aggregated to the `cm cache` folder. The following command could be used to browse the structure of inference results folder generated by CM.
+    ### Get results folder structure
+    ```bash
+    cm find cache --tags=get,mlperf,inference,results,dir | xargs tree
```
Comment (anandhu-eng) [inline docs/submission/index.md:10]: Should we create a new script in CM for the purpose of showing the directory structure using tree?

Comment (arjunsuresh) [reply docs/submission/index.md:10]: @anandhu-eng what's the advantage?

Comment (anandhu-eng) [reply docs/submission/index.md:10]: We could prevent the usage of commands other than CM in documentation. I don't find any specific advantage.

Comment (arjunsuresh) [reply docs/submission/index.md:10]: Thats not necessary. `tree` command is fine to be used.

**Review by arjunsuresh — APPROVED** [review]

**Pull request merged**